src/main/java/com/vasy/security/dto/UserDetailDto.java

```diff
@@ -0,0 +1,13 @@
+package com.vasy.security.dto;
+
+import lombok.Data;
+
+@Data
+public class UserDetailDto {
+	
+	private long id;	
```

Comment:
I Id is auto generated then why are you taking as input?

Reply:
I'm not taking id as input just passing in the response.